web/frontend/src/components/components.module.css

```diff
@@ -74,7 +74,7 @@
 .FeedbackButtonDiv{
     position:absolute;
     top:50%;
-    right:-1%;
+    right:0;
```

Comment:
-0.6% looks good , tried for phase mapper also

Reply:
I used the transform-origin to get a better rotation.